### ISSUE TITLE
error: cannot access ActivityCompatApi23

### ISSUE THREAD

**Issue by JoaoArmandoo** [HendraAnggrian/reveallayout#4]
Hello, first of all congratulations for the extension, and thank you for sharing.
I'm facing some problems with the library, yesterday I had the same problem, however it was solved when I reduced the versions of the suport librarys, (I just think that was it). The problem happened again today, but this time, I could not solve it, even reducing the versions of the library, removing them the problem continues unfortunately. I would like your help to be able to repair if possible, again thank you!
![image](https://user-images.githubusercontent.com/28633653/29301265-61f26562-8151-11e7-8d0e-5217eca7ab16.png)

its my gradle 

`buildscript {
    repositories {
        maven { url 'https://maven.fabric.io/public' }
    }

    dependencies {
        classpath 'io.fabric.tools:gradle:1.+'

    }
}
apply plugin: 'com.android.application'
apply plugin: 'io.fabric'

repositories {
    maven {
        url 'https://maven.fabric.io/public'
    }

    maven { url "https://jitpack.io" }
}

android {
    compileSdkVersion 25
    buildToolsVersion "25.0.3"
    defaultConfig {
        applicationId "joaoarmando.com.jaap"
        minSdkVersion 16
        targetSdkVersion 25
        multiDexEnabled true
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    packagingOptions {
        exclude 'META-INF/DEPENDENCIES'
        exclude 'META-INF/NOTICE'
        exclude 'META-INF/LICENSE'
        exclude 'META-INF/license.txt'
        exclude 'META-INF/notice.txt'
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }

}

dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile files('libs/twitter4j-core-4.0.4.jar')
    compile files('libs/jInstagram-1.2.1.jar')
    // https://mvnrepository.com/artifact/org.slf4j/slf4j-simple
    // https://mvnrepository.com/artifact/joda-time/joda-time
    compile('com.twitter.sdk.android:twitter:2.3.2@aar') {
        transitive = true;
    }
//    compile ('com.github.ozodrukh:CircularReveal:2.0.1@aar') {
//        transitive = true;
//    }
    compile 'org.slf4j:slf4j-simple:1.6.1'
    compile 'joda-time:joda-time:2.3'
    compile 'com.github.kanytu:android-parallax-listview:v1.1'
    compile 'com.github.vajro:MaterialDesignLibrary:1.6'
    compile 'com.github.clans:fab:1.6.4'
//    compile 'com.google.firebase:firebase-core:9.6.1'
    compile 'com.facebook.android:facebook-android-sdk:[4,5)'
    compile 'com.google.code.gson:gson:2.7'
    compile 'com.squareup.picasso:picasso:2.5.2'
    compile 'it.neokree:MaterialTabs:0.11'
    compile 'org.apache.commons:commons-lang3:3.5'
    compile 'com.ogaclejapan.smarttablayout:library:1.6.1@aar'
    compile 'com.ogaclejapan.smarttablayout:utils-v4:1.6.1@aar'
    compile 'com.wrapp.floatlabelededittext:library:0.0.6'
    compile 'com.nineoldandroids:library:2.4.0'
    compile 'org.solovyev.android.views:linear-layout-manager:0.5@aar'
    compile 'com.rengwuxian.materialedittext:library:2.1.4'
    compile 'com.weiwangcn.betterspinner:library-material:1.1.0'
    compile 'com.parse:parse-android:1.15.8'
    compile 'com.parse:parsefacebookutils-v4-android:1.10.4@aar'
    compile 'com.squareup:otto:1.3.8'
    compile 'com.android.support:appcompat-v7:23.0.1'
    compile 'com.android.support:design:23.1.1'
    compile 'com.android.support:support-v4:23.0.1'
    compile 'com.hendraanggrian:reveallayout:0.5.3'

    testCompile 'junit:junit:4.12'
}
apply plugin: 'com.google.gms.google-services'

`

**Comment by JoaoArmandoo** [comment]
update: I just downgraded the library version, tried with 0.5.2, but the problem persisted, now at 0.5.1 the problem apparently disappeared.

**Comment by HendraAnggrian** [comment]
Hi @JoaoArmandoo.

Since version 0.5.2, it uses support library version 26.0.0, which is only available from Google's maven repository. Try updating your repository list:

```gradle
repositories {
    maven { url 'https://maven.fabric.io/public' }
    maven { url 'https://maven.google.com' }
    maven { url 'https://jitpack.io' }
}
```

If you have no intention of using support library version 26.0.0, then version 0.5.1 should work just fine.

**Comment by JoaoArmandoo** [comment]
I tried this, but it did not solve the problem, I will close this, since it was solved reducing to version 0.5.1 and it already catches me perfectly well.

But before, being grateful for your attention, could you give me more information on how to do the second animation? I did not understand very well. Thank you!
[https://github.com/HendraAnggrian/reveallayout/blob/master/art/demo2.gif](url)

**Comment by HendraAnggrian** [comment]
You might also need to update your `compileSdkVersion` and `targetSdkVersion` to 26. And while you're in there, also try `buildToolsVersion` 26.0.1.

Regarding the second animation, it is a combination of circular reveal and path animation. It is still pretty unstable: it only expected to work from down to up direction. Supporting other directions would require better understanding of how this path animation works, which is too bad since I've invested very little time to this library.
